src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/RectangleSelectionRenderer.js

```diff
@@ -6,71 +6,42 @@ import PositionNormalizer from './normalizers/PositionNormalizer';
 import RatioNormalizer from './normalizers/RatioNormalizer';
 import RoundingNormalizer from './normalizers/RoundingNormalizer';
 import SizeNormalizer from './normalizers/SizeNormalizer';
+import withPercentageValues from './withPercentageValues';
 
 type Props = {
     backdrop: boolean,
     containerHeight: number,
     containerWidth: number,
     disabled: boolean,
-    forceRatio: boolean,
     label?: string,
-    minHeight?: number,
+    minHeight: number | typeof undefined,
     minSizeNotification: boolean,
-    minWidth?: number,
+    minWidth: number | typeof undefined,
     onChange: (s: ?SelectionData) => void,
     round: boolean,
     usePercentageValues: boolean,
```

Comment:
This is probably not needed anymore in here, right?

Reply:
Same as above